samples/bluetooth/peripheral_esl/README.rst

```diff
@@ -0,0 +1,264 @@
+For now, only :ref:`nrf52dk_nrf52832` and :ref:`nrf52833dk_nrf52833` support this optional feature.
+In VS Code with nRF Extension, select prj_release.conf and select “nrf52833dk_nrf52833_power_profiler_epd.conf” as Kconfig fragments.
+As alternative, using terminal, to activate the optional feature supported by this sample, use the following build command:
+
+   .. code-block:: console
+
+      west build -p -b nrf52833dk_nrf52833 -- -DOVERLAY_CONFIG=nrf52833dk_nrf52833_power_profiler_release_epd.conf -DDTC_OVERLAY_FILE=conf/nrf52833dk_nrf52833/nrf52833dk_nrf52833_release_epd.overlay
+
+See :ref:`cmake_options` for instructions on how to add this option.
+For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
+
+.. note::
+
+   Please refer to the original manufacturer's documentation of the panel for selecting the shield board switch positions. For example, adjust to A0 for `WAVESHARE 250x122, 2.13inch E-Ink raw display panel`.
+
+.. _peripheral_esl_power_profiling_variant:
+
+Power Profiling variant
+=======================
+
+Power optimized overlay disables all of unnecessary peripheral to minimize power consumption. You can use power profiler overlay for :ref:`nrf52833dk_nrf52833` to measure current. In VS Code with nRF Extension, select prj_release.conf and select “nrf52833dk_nrf52833_power_profiler_release_led.conf” as Kconfig fragments.
+As alternative, using terminal, use the following build command:
+
+   .. code-block:: console
+
+      west build -p -b nrf52833dk_nrf52833 -- -DCONF_FILE=prj_release.conf -DOVERLAY_CONFIG=nrf52833dk_nrf52833_power_profiler_release_led.conf -DDTC_OVERLAY_FILE=conf/nrf52833dk_nrf52833/nrf52833dk_nrf52833_power_profiler_release.overlay
+
+Please follow  :ref:`peripheral_esl_power_measure` to measure currents.
+
+.. _peripheral_esl_power_measure:
+
+Power Consumption Measurement
+=============================
+
+Prerequisite: Read :ref:`app_power_opt_general` and get `Power Profiler Kit II (PPK2)`_
+
+1. Build and program the Tag (:ref:`nrf52833dk_nrf52833`) Power Profiling configuration by following :ref:`peripheral_esl_build_and_program` and :ref:`peripheral_esl_power_profiling_variant`
+
+#. Follow `Preparing a DK for current measurement`_ to prepare 52833DK for Power Consumption measurements
+
+   a. Cut SB40.
+   b. Connect VOUT of PPK2 to nRF current Measurement on DK, connect GND of PPK2 to External supply negative polarity pin (`-`). By doing so, PPK2 will provide power to only the nRF52 SoC.
+   c. Connect the PPK2 (powered ON) in “Source Meter” mode to the :ref:`nrf52833dk_nrf52833` (turned ON) and enable power output 3000mV on the Power Profiler GUI and start measurement. Power the rest of the ESL Tag DevKit via USB (same as programming USB port), and keep the DevKit turned ON.
```

Comment:
It seems to me that in the sample dokumentation we don't need a dedscription of how to use the Power Profiler Kit. This is included in the PPK documentation.